src/components/common/Semaphore/index.tsx

```diff
@@ -0,0 +1,34 @@
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import React from 'react';
+
+type SemaphoreProps = {
+    value: string | number | boolean;
+};
+
+export const Semaphore = (props: SemaphoreProps) => {
+    const { value } = props;
+    if (typeof value === 'number') {
+        if (value > 4) {
```

Comment:
this will need refactoring since this is based on a 2D grid really and not a range. We may need to refactor this later in a diff PR once the TVL calc is incorporated